src/components/networks/networkOverviewTab.jsx

```diff
@@ -53,7 +56,47 @@ const DHCPHost = (host, index, family, idPrefix) => {
 
     const hostInfo = hostVals.join(", ");
 
-    return <ListItem key={index} id={id}> {hostInfo} </ListItem>;
+    const removeDHCPHostButton = (
+        <DeleteResourceButton objectId={`${id}-button`}
+            actionName={_("remove")}
+            variant="link"
+            isInline
+            showDialog={() => setDeleteDialogProps({
+                objectType: _("static host from DHCP"),
+                objectName: "",
+                actionDescription: _(`The static host entry for ${host.ip} will be removed:`),
```

Comment:
This does not work for i18n, as the gettext call happens after the string mutation. You need to use cockpit.format() for this.

    git grep '_(`'

This ^ shows two other instances of this, can you please fix them in a separate commit or PR?

Reply:
I filed PR #655 to fix and spot that.